visualization/main.py

```diff
@@ -70,6 +71,7 @@ def _get_line(
         color="red",
     )
     p.xaxis.major_label_orientation = "vertical"
+    p.y_range = Range1d(0, (int(max(other_qm_ys)) * 1.1))
```

Comment:
Do you have an example of what the graphs look like with and without multiplying by 10%?

Reply:
![image](https://user-images.githubusercontent.com/66801011/233596423-16a1a19d-ef50-4c5c-ad2e-9dd053e7538a.png)

Example with.

without buffer the line would follow the "roof" of the graph

Reply:
I'm open to removing this "magic" number, I will just merge now for the sake of the experiment such that I get graphs in bounds.